src/protocol/attribution/mod.rs

```diff
@@ -162,77 +162,52 @@ where
 /// ## Panics
 /// Nah, it doesn't.
 ///
-pub async fn do_the_binary_tree_thing<'a, F, C, S, I>(
+pub async fn do_the_binary_tree_thing<F, C, S>(
     ctx: C,
-    helper_bits: &[S],
-    credits: I,
+    stop_bits: &[S],
+    values: &mut [S],
```

Comment:
Renamed to `values` since we can probably use this logic to accumulate time deltas for lookback window protocol.